src/Aspire.Hosting/DistributedApplication.cs

```diff
@@ -143,6 +143,7 @@ private void WriteStartingLog()
         }
 
         _logger.LogInformation("Distributed application starting.");
+        _logger.LogInformation("Working directory is: {WorkingDirectory}", Directory.GetCurrentDirectory());
```

Comment:
I actually now think instead of this, lets log the AppHostDirectory

https://github.com/dotnet/aspire/blob/5c76a198c8409f1a8c7a39d1ae10c81094a317e4/src/Aspire.Hosting/DistributedApplicationBuilder.cs#L51

That's what all of the paths are relative to by default, so maybe that's more useful than working directory.